lib/libicp/Makefile.am

```diff
@@ -31,7 +31,6 @@ nodist_libicp_la_SOURCES = \
 	module/icp/illumos-crypto.c \
 	module/icp/io/aes.c \
 	module/icp/io/sha2_mod.c \
-	module/icp/io/skein_mod.c \
```

Comment:
shouldn't this removal of skein_mod.c go in the other commit?

Reply:
Good spot, fixed.